client/src/components/Collections/common/PairingFilterInputGroup.vue

```diff
@@ -0,0 +1,65 @@
+<script setup lang="ts">
+import { BDropdown, BDropdownItem, BFormInput, BInputGroup } from "bootstrap-vue";
+import { ref, watch } from "vue";
+
+import { COMMON_FILTERS } from "@/components/Collections/pairing";
+
+interface Props {
+    forwardFilter: string;
+    reverseFilter: string;
+}
+
+const emit = defineEmits<{
+    (e: "on-update", forwardFilter: string, reverseFilter: string): void;
+}>();
+
+function update(forward: string, reverse: string) {
+    emit("on-update", forward, reverse);
+}
+
+const props = defineProps<Props>();
+
+const currentForwardFilter = ref(props.forwardFilter);
+const currentReverseFilter = ref(props.reverseFilter);
+
+watch(
+    () => props.forwardFilter,
+    () => {
+        currentForwardFilter.value = props.forwardFilter;
+    }
+);
+watch(
+    () => props.reverseFilter,
+    () => {
+        currentReverseFilter.value = props.reverseFilter;
+    }
+);
+
+function resync() {
+    emit("on-update", currentForwardFilter.value, currentReverseFilter.value);
+}
+
+watch(currentForwardFilter, resync);
+watch(currentForwardFilter, resync);
```

Comment:
Did you want this to be 
```suggestion
watch(currentReverseFilter, resync);
```
?

Reply:
I suspect yes - rebased with change.